{"created": 1733060031.0666082, "duration": 3.400456428527832, "exitcode": 0, "root": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite", "environment": {}, "summary": {"passed": 6, "total": 6, "collected": 6}, "tests": [{"nodeid": "tests/test_posts.py::TestPostAPI::test_get_posts", "lineno": 9, "outcome": "passed", "keywords": ["test_get_posts", "posts_tests", "pytestmark", "TestPostAPI", "test_posts.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.001086411999494885, "outcome": "passed", "longrepr": "[gw1] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 1.4241658809987712, "outcome": "passed", "log": [{"name": "test_posts", "msg": "Starting test: test_get_posts", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 28, "funcName": "test_get_posts", "created": 1733060028.5998237, "msecs": 599.0, "relativeCreated": 784.3787670135498, "thread": 129493719175168, "threadName": "MainThread", "processName": "MainProcess", "process": 17650, "asctime": "2024-12-01 19:03:48,599"}, {"name": "test_posts", "msg": "GET https://jsonplaceholder.typicode.com/posts - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 32, "funcName": "test_get_posts", "created": 1733060029.7999518, "msecs": 799.0, "relativeCreated": 1984.5068454742432, "thread": 129493719175168, "threadName": "MainThread", "processName": "MainProcess", "process": 17650, "asctime": "2024-12-01 19:03:49,799"}, {"name": "test_posts", "msg": "Response contains 100 items", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 38, "funcName": "test_get_posts", "created": 1733060029.80021, "msecs": 800.0, "relativeCreated": 1984.7650527954102, "thread": 129493719175168, "threadName": "MainThread", "processName": "MainProcess", "process": 17650, "asctime": "2024-12-01 19:03:49,800"}, {"name": "test_posts", "msg": "All items validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 44, "funcName": "test_get_posts", "created": 1733060030.023386, "msecs": 23.0, "relativeCreated": 2207.9410552978516, "thread": 129493719175168, "threadName": "MainThread", "processName": "MainProcess", "process": 17650, "asctime": "2024-12-01 19:03:50,023"}, {"name": "test_posts", "msg": "Finished test: test_get_posts", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 59, "funcName": "test_get_posts", "created": 1733060030.0234725, "msecs": 23.0, "relativeCreated": 2208.0276012420654, "thread": 129493719175168, "threadName": "MainThread", "processName": "MainProcess", "process": 17650, "asctime": "2024-12-01 19:03:50,023"}], "longrepr": "[gw1] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.000186569999641506, "outcome": "passed", "longrepr": "[gw1] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}, {"nodeid": "tests/test_comments.py::TestCommentsAPI::test_get_comments", "lineno": 9, "outcome": "passed", "keywords": ["test_get_comments", "comments_tests", "pytestmark", "TestCommentsAPI", "test_comments.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.0011016019998351112, "outcome": "passed", "longrepr": "[gw0] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 2.465083016999415, "outcome": "passed", "log": [{"name": "test_comments", "msg": "Starting test: test_get_comments", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "filename": "test_comments.py", "module": "test_comments", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 28, "funcName": "test_get_comments", "created": 1733060028.5999281, "msecs": 599.0, "relativeCreated": 815.6986236572266, "thread": 137133981392896, "threadName": "MainThread", "processName": "MainProcess", "process": 17647, "asctime": "2024-12-01 19:03:48,599"}, {"name": "test_comments", "msg": "GET https://jsonplaceholder.typicode.com/comments - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "filename": "test_comments.py", "module": "test_comments", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 32, "funcName": "test_get_comments", "created": 1733060029.75767, "msecs": 757.0, "relativeCreated": 1973.440408706665, "thread": 137133981392896, "threadName": "MainThread", "processName": "MainProcess", "process": 17647, "asctime": "2024-12-01 19:03:49,757"}, {"name": "test_comments", "msg": "Response contains 500 comments", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "filename": "test_comments.py", "module": "test_comments", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 39, "funcName": "test_get_comments", "created": 1733060029.7595997, "msecs": 759.0, "relativeCreated": 1975.370168685913, "thread": 137133981392896, "threadName": "MainThread", "processName": "MainProcess", "process": 17647, "asctime": "2024-12-01 19:03:49,759"}, {"name": "test_comments", "msg": "All comments validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "filename": "test_comments.py", "module": "test_comments", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 44, "funcName": "test_get_comments", "created": 1733060031.064212, "msecs": 64.0, "relativeCreated": 3279.982566833496, "thread": 137133981392896, "threadName": "MainThread", "processName": "MainProcess", "process": 17647, "asctime": "2024-12-01 19:03:51,064"}, {"name": "test_comments", "msg": "Finished test: test_get_comments", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "filename": "test_comments.py", "module": "test_comments", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 59, "funcName": "test_get_comments", "created": 1733060031.064342, "msecs": 64.0, "relativeCreated": 3280.1125049591064, "thread": 137133981392896, "threadName": "MainThread", "processName": "MainProcess", "process": 17647, "asctime": "2024-12-01 19:03:51,064"}], "longrepr": "[gw0] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.00019738100127142388, "outcome": "passed", "longrepr": "[gw0] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}, {"nodeid": "tests/test_posts.py::TestPostAPI::test_get_post", "lineno": 60, "outcome": "passed", "keywords": ["test_get_post", "posts_tests", "pytestmark", "TestPostAPI", "test_posts.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.0010544559991103597, "outcome": "passed", "longrepr": "[gw2] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 0.46191811799872085, "outcome": "passed", "log": [{"name": "test_posts", "msg": "Starting test: test_get_post", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 79, "funcName": "test_get_post", "created": 1733060028.5998273, "msecs": 599.0, "relativeCreated": 749.0851879119873, "thread": 136928575406080, "threadName": "MainThread", "processName": "MainProcess", "process": 17653, "asctime": "2024-12-01 19:03:48,599"}, {"name": "test_posts", "msg": "GET https://jsonplaceholder.typicode.com/posts/2 - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 83, "funcName": "test_get_post", "created": 1733060029.057765, "msecs": 57.0, "relativeCreated": 1207.0229053497314, "thread": 136928575406080, "threadName": "MainThread", "processName": "MainProcess", "process": 17653, "asctime": "2024-12-01 19:03:49,057"}, {"name": "test_posts", "msg": "Response data ID: 2 matches the expected value", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 90, "funcName": "test_get_post", "created": 1733060029.057929, "msecs": 57.0, "relativeCreated": 1207.1869373321533, "thread": 136928575406080, "threadName": "MainThread", "processName": "MainProcess", "process": 17653, "asctime": "2024-12-01 19:03:49,057"}, {"name": "test_posts", "msg": "Post data validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 94, "funcName": "test_get_post", "created": 1733060029.061079, "msecs": 61.0, "relativeCreated": 1210.3369235992432, "thread": 136928575406080, "threadName": "MainThread", "processName": "MainProcess", "process": 17653, "asctime": "2024-12-01 19:03:49,061"}, {"name": "test_posts", "msg": "Finished test: test_get_post", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "filename": "test_posts.py", "module": "test_posts", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 109, "funcName": "test_get_post", "created": 1733060029.0611725, "msecs": 61.0, "relativeCreated": 1210.430383682251, "thread": 136928575406080, "threadName": "MainThread", "processName": "MainProcess", "process": 17653, "asctime": "2024-12-01 19:03:49,061"}], "longrepr": "[gw2] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.00023109699941414874, "outcome": "passed", "longrepr": "[gw2] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}, {"nodeid": "tests/test_users.py::TestUsersAPI::test_fetch_single_user", "lineno": 61, "outcome": "passed", "keywords": ["test_fetch_single_user", "users_tests", "pytestmark", "TestUsersAPI", "test_users.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.0010818869996001013, "outcome": "passed", "longrepr": "[gw4] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 0.7086969000010868, "outcome": "passed", "log": [{"name": "test_users", "msg": "Starting test: test_fetch_single_user", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 81, "funcName": "test_fetch_single_user", "created": 1733060028.6000094, "msecs": 600.0, "relativeCreated": 679.1417598724365, "thread": 123593530613760, "threadName": "MainThread", "processName": "MainProcess", "process": 17659, "asctime": "2024-12-01 19:03:48,600"}, {"name": "test_users", "msg": "GET https://jsonplaceholder.typicode.com/users/1 - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 85, "funcName": "test_fetch_single_user", "created": 1733060029.2994785, "msecs": 299.0, "relativeCreated": 1378.6108493804932, "thread": 123593530613760, "threadName": "MainThread", "processName": "MainProcess", "process": 17659, "asctime": "2024-12-01 19:03:49,299"}, {"name": "test_users", "msg": "User data contains expected keys: dict_keys(['id', 'name', 'username', 'email', 'address', 'phone', 'website', 'company'])", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 92, "funcName": "test_fetch_single_user", "created": 1733060029.2996392, "msecs": 299.0, "relativeCreated": 1378.7715435028076, "thread": 123593530613760, "threadName": "MainThread", "processName": "MainProcess", "process": 17659, "asctime": "2024-12-01 19:03:49,299"}, {"name": "test_users", "msg": "User data validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 96, "funcName": "test_fetch_single_user", "created": 1733060029.308074, "msecs": 308.0, "relativeCreated": 1387.2063159942627, "thread": 123593530613760, "threadName": "MainThread", "processName": "MainProcess", "process": 17659, "asctime": "2024-12-01 19:03:49,308"}, {"name": "test_users", "msg": "Finished test: test_fetch_single_user", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 111, "funcName": "test_fetch_single_user", "created": 1733060029.308155, "msecs": 308.0, "relativeCreated": 1387.2873783111572, "thread": 123593530613760, "threadName": "MainThread", "processName": "MainProcess", "process": 17659, "asctime": "2024-12-01 19:03:49,308"}], "longrepr": "[gw4] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.00021431700042739976, "outcome": "passed", "longrepr": "[gw4] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}, {"nodeid": "tests/test_users.py::TestUsersAPI::test_search_for_user", "lineno": 112, "outcome": "passed", "keywords": ["test_search_for_user", "users_tests", "pytestmark", "TestUsersAPI", "test_users.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.0011097220012743492, "outcome": "passed", "longrepr": "[gw6] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 0.6733141919994523, "outcome": "passed", "log": [{"name": "test_users", "msg": "Starting test: test_search_for_user", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 131, "funcName": "test_search_for_user", "created": 1733060028.600001, "msecs": 600.0, "relativeCreated": 611.6979122161865, "thread": 137947486412800, "threadName": "MainThread", "processName": "MainProcess", "process": 17666, "asctime": "2024-12-01 19:03:48,600"}, {"name": "test_users", "msg": "GET https://jsonplaceholder.typicode.com//users?id=2 - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 135, "funcName": "test_search_for_user", "created": 1733060029.2602193, "msecs": 260.0, "relativeCreated": 1271.916151046753, "thread": 137947486412800, "threadName": "MainThread", "processName": "MainProcess", "process": 17666, "asctime": "2024-12-01 19:03:49,260"}, {"name": "test_users", "msg": "Search result contains 1 user(s)", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 142, "funcName": "test_search_for_user", "created": 1733060029.260649, "msecs": 260.0, "relativeCreated": 1272.345781326294, "thread": 137947486412800, "threadName": "MainThread", "processName": "MainProcess", "process": 17666, "asctime": "2024-12-01 19:03:49,260"}, {"name": "test_users", "msg": "User search result validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 147, "funcName": "test_search_for_user", "created": 1733060029.27266, "msecs": 272.0, "relativeCreated": 1284.3568325042725, "thread": 137947486412800, "threadName": "MainThread", "processName": "MainProcess", "process": 17666, "asctime": "2024-12-01 19:03:49,272"}, {"name": "test_users", "msg": "Finished test: test_search_for_user", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 162, "funcName": "test_search_for_user", "created": 1733060029.272749, "msecs": 272.0, "relativeCreated": 1284.4457626342773, "thread": 137947486412800, "threadName": "MainThread", "processName": "MainProcess", "process": 17666, "asctime": "2024-12-01 19:03:49,272"}], "longrepr": "[gw6] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.00022275099945545662, "outcome": "passed", "longrepr": "[gw6] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}, {"nodeid": "tests/test_users.py::TestUsersAPI::test_fetch_all_users", "lineno": 9, "outcome": "passed", "keywords": ["test_fetch_all_users", "users_tests", "pytestmark", "TestUsersAPI", "test_users.py", "tests", "JsonPlaceholder-API-Automation-Suite", ""], "setup": {"duration": 0.001144869000199833, "outcome": "passed", "longrepr": "[gw3] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "call": {"duration": 0.7482559319996653, "outcome": "passed", "log": [{"name": "test_users", "msg": "Starting test: test_fetch_all_users", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 29, "funcName": "test_fetch_all_users", "created": 1733060028.60003, "msecs": 600.0, "relativeCreated": 717.5476551055908, "thread": 125084069838848, "threadName": "MainThread", "processName": "MainProcess", "process": 17656, "asctime": "2024-12-01 19:03:48,600"}, {"name": "test_users", "msg": "GET https://jsonplaceholder.typicode.com/users - Status Code: 200", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 33, "funcName": "test_fetch_all_users", "created": 1733060029.2549832, "msecs": 254.0, "relativeCreated": 1372.5008964538574, "thread": 125084069838848, "threadName": "MainThread", "processName": "MainProcess", "process": 17656, "asctime": "2024-12-01 19:03:49,254"}, {"name": "test_users", "msg": "Response contains 10 users", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 40, "funcName": "test_fetch_all_users", "created": 1733060029.2555945, "msecs": 255.0, "relativeCreated": 1373.1122016906738, "thread": 125084069838848, "threadName": "MainThread", "processName": "MainProcess", "process": 17656, "asctime": "2024-12-01 19:03:49,255"}, {"name": "test_users", "msg": "All users validated successfully against the schema", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 45, "funcName": "test_fetch_all_users", "created": 1733060029.3475606, "msecs": 347.0, "relativeCreated": 1465.078353881836, "thread": 125084069838848, "threadName": "MainThread", "processName": "MainProcess", "process": 17656, "asctime": "2024-12-01 19:03:49,347"}, {"name": "test_users", "msg": "Finished test: test_fetch_all_users", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "filename": "test_users.py", "module": "test_users", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 60, "funcName": "test_fetch_all_users", "created": 1733060029.3476927, "msecs": 347.0, "relativeCreated": 1465.2104377746582, "thread": 125084069838848, "threadName": "MainThread", "processName": "MainProcess", "process": 17656, "asctime": "2024-12-01 19:03:49,347"}], "longrepr": "[gw3] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}, "teardown": {"duration": 0.00020350300110294484, "outcome": "passed", "longrepr": "[gw3] linux -- Python 3.10.12 /home/ss/pyenv/bin/python3"}}], "warnings": [{"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}, {"message": "Unknown config option: comments_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: posts_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown config option: users_tests\n", "category": "PytestConfigWarning", "when": "config", "filename": "/home/ss/pyenv/lib/python3.10/site-packages/_pytest/config/__init__.py", "lineno": 1441}, {"message": "Unknown pytest.mark.comments_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_comments.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 10}, {"message": "Unknown pytest.mark.posts_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_posts.py", "lineno": 61}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 10}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 62}, {"message": "Unknown pytest.mark.users_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/home/ss/AutoTestLab/JsonPlaceholder-API-Automation-Suite/tests/test_users.py", "lineno": 113}]}